lib/scr/screens/auth/auth_screen.dart

```diff
@@ -41,37 +38,70 @@ class AuthScreen extends StatelessWidget {
             SizedBox(height: 32),
 
             Padding(
-              padding: const EdgeInsets.symmetric(horizontal: 16),
+              padding: AppPadding.horizontal,
               child: CupertinoButton(
                 padding: const EdgeInsets.symmetric(vertical: 20),
-                color: Color(0xFF4631D2),
+                color: AppColors.main,
                 child: Text('Войти',
                         style: TextStyle(
                           fontWeight: FontWeight.bold
                         ),
                       ), 
-                onPressed: (){}
-               ),
+                onPressed: (){
+                  Navigator.push(
```

Comment:
Тут можно заменить код из `.push` на `.pushNamed`, т.к. у вас есть роутер:
```suggestion
                  Navigator.pushNamed(context, MainRoute);
```